prow/monitoring/prometheus_operator/Makefile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: prow-monitoring-deploy
+
+.PHONY : prow-monitoring-pre-steps
+prow-monitoring-pre-steps:
+	@echo "running prow-monitoring-pre-steps ..."
+	kubectl apply -f prow_monitoring_namespace.yaml
```

Comment:
https://github.com/kubernetes/test-infra/blob/4dbd419a2ea8d53036d447c3b63f92a3306fce33/prow/cluster/BUILD.bazel#L11

references

https://github.com/kubernetes/test-infra/blob/4dbd419a2ea8d53036d447c3b63f92a3306fce33/prow/cluster/build_rbac.yaml

Reply:
Also you can follow the trail to figure out how things like component work and are defined:

https://github.com/kubernetes/test-infra/blob/4dbd419a2ea8d53036d447c3b63f92a3306fce33/prow/cluster/BUILD.bazel#L3

https://github.com/kubernetes/test-infra/blob/4dbd419a2ea8d53036d447c3b63f92a3306fce33/prow/def.bzl#L132

https://github.com/kubernetes/test-infra/blob/4dbd419a2ea8d53036d447c3b63f92a3306fce33/prow/def.bzl#L105-L106

https://github.com/kubernetes/test-infra/blob/4dbd419a2ea8d53036d447c3b63f92a3306fce33/prow/def.bzl#L19

AKA its all just "convenience" wrappers around https://github.com/bazelbuild/rules_k8s calls